Docs/Reports/Deliverables/D5.1/references.bib

```diff
@@ -85,4 +84,193 @@ @Misc{Fortmann-Grote:zenodo2019
   doi    = {10.5281/zenodo.3524710},
 }
 
+@Misc{Juncheng:zenodo2019,
+  author = {E, Juncheng},
+  title  = {Example dataset for openPMD-conformed molecular dynamics data (MD domain extension)},
+  year   = {2019},
+  date   = {2019-11-02},
+  doi    = {10.5281/zenodo.3524710},
```

Comment:
10.5281/zenodo.3524710: That's my WPG dataset. Please submit your own and change the reference.